README.md

```diff
@@ -16,6 +16,42 @@ equally likely, i.e. the input array is ordered randomly.
 Your answer must derive probabilities for choosing a good pivot and
 quantitatively reason with them.
 
+The intuitive answer I immediately came to is that yes, median of three is more likely to produce a good pivot than selecting the first element with no further information. It is pretty easy to understand this on a base level, as worst case scenario for quicksort is when the selected pivot is the highest or lowest available element. By starting off with two comparisons you guarantee that there is at least one element higher than your pivot and at least one element lower than your pivot, which eliminates both worst case scenarios. 
+
+Diving Deeper (With help from Daniel Collins and trusty Slide 34):
```

Comment:
So to what extent is this your work and what was done by Daniel?

Reply:
Daniel directed me to the idea of thinking about our median of 3 values in terms of combinations, and then evaluating those combinations in terms of probabilities. Admittedly this is the difficult part of the solution in my opinion, but I still checked all combination/permutations and calculated their probabilities myself, giving my own explanations throughout to show understanding.

Reply:
The concept of how to go about this problem did not come to me spontaneously no, however I would still argue that applying that approach to the problem was the bulk of the 'work' which I did myself. As I said before the difficult part was the approach and I acknowledge that my own research and knowledge last night did not get me to the conclusion alone, but as with all problems where I seek out aid externally, I tried to make an effort to not be given the answer, and rather have help with approaches where needed and apply and explain them myself. However I do of course understand if that is not sufficient for this particular exercise.

Reply:
I'll make an exception in this case.